core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/scheduling/ExpansionGreedyScheduleGenerator.scala

```diff
@@ -152,35 +160,207 @@ class ExpansionGreedyScheduleGenerator(
     Left(regionDAG)
   }
 
-  private def handleDependentLinks(
+  /**
+    * A dependee input port is one that is depended on by another input port of the same operator.
+    * The incoming edge of a dependee input port is called a dependee edge.
```

Comment:
try to make documentation concise. i feel a few docs are way too long to get the point.